bindings/c/test/apitester/tests/CApiBlobGranuleCorrectnessBlocking.toml

```diff
@@ -22,5 +22,6 @@ maxKeyLength = 64
 minValueLength = 1
 maxValueLength = 1000
 maxKeysPerTransaction = 50
+# TODO - increase initialSize and/or buggify down BG_SNAPSHOT_FILE_TARGET_BYTES to force multiple granules
```

Comment:
We can increase the initialSize if necessary. I think in some other tests it was not relevant or even desired that there is some more chance of having multiple transactions touching the same key.

Reply:
Makes sense. One of my concerns was if it significantly increased the runtime of the tests that might be undesirable, we can validate that though.